File: dutils/ilclasses.cs
using System;
namespace HelloApp
{   class GetHello { public static string secretmessage(){ string hello = "Hello world!"; string result = "";foreach (char element in hello){result = result + element;}return result;}} class DoWrite{public static void arn(string input) => Console.WriteLine(input);}class Program{static void Main(string[] args){string smm = GetHello.secretmessage();DoWrite.arn(smm);}}}

File: dutils/ilclasses.cpp
#include <iostream>
using namespace std;int main(void) {struct hello {string m = "Hello World";string out = m; };class h {public: int i = 0;struct hello fuck;h(void) {};void print(void) {cout << fuck.out << endl;}};class e : public h {public: e(void) {i++;};};class l : public e {public: l(void) {i++;};};class o : public l {public: o(void) {i++;};};o hell;hell.print();return 0;
}